pkg/storage/bloom/v1/bounds.go

```diff
@@ -17,6 +20,34 @@ const (
 	After
 )
 
+func ParseFingerprint(s string) (model.Fingerprint, error) {
+	fp, err := strconv.ParseUint(s, 16, 64)
+	if err != nil {
+		return 0, fmt.Errorf("error parsing fingerprint %s : %w", s, err)
+	}
+	return model.Fingerprint(fp), nil
+}
```

Comment:
The package `github.com/prometheus/common/model` would also provide this functionality.